.gitignore

```diff
@@ -21,6 +21,10 @@
 /.build
 /Packages
 
+# Commonly used name for build products for llbuild-analyze.
+/products/build
```

Comment:
Why is this one here?

Reply:
It got created when building llbuild-analyze in the IDE. We could either re-configure that or ignore the `build` dir, I decided to ignore it. Would you suggest to do the former?